cpp/server/ct-server.cc

```diff
@@ -92,7 +96,6 @@ static const bool cert_dummy =
 
 }  // namespace
 
-
```

Comment:
Please leave the empty line. Have you been using `clang-format`? There's a configuration file for it in the repository, by the way.